src/server/editorServices.ts

```diff
@@ -764,33 +885,27 @@ module ts.server {
                 return { errorMsg: "tsconfig syntax error" };
             }
             else {
-                // REVIEW: specify no base path so can get absolute path below
-                var parsedCommandLine = ts.parseConfigFile(rawConfig);
-
-                if (parsedCommandLine.errors) {
-                    // TODO: gather diagnostics and transmit
+                var parsedCommandLine = ts.parseConfigFile(rawConfig, dirPath);
+                if (parsedCommandLine.errors && (parsedCommandLine.errors.length > 0)) {
                     return { errorMsg: "tsconfig option errors" };

```

Comment:
do we want to kill the whole process here cause there is an unknown configuration option, or should we just use what we have?